addon/-private/system/references/belongs-to.js

```diff
@@ -63,32 +63,33 @@ BelongsToReference.prototype.push = function(objectOrPromise) {
   });
 };
 
-BelongsToReference.prototype.value = function() {
-  var inverseRecord = this.belongsToRelationship.inverseRecord;
+BelongsToReference.prototype.isLoaded = function() {
+  let inverseRecord = this.belongsToRelationship.inverseRecord;
+
+  return inverseRecord && inverseRecord.isLoaded();
```

Comment:
This seems to miss the case where it is null. If we know we are null, are we loaded?